bokehjs/src/lib/models/text/mathjax/index.ts

```diff
@@ -12,19 +12,26 @@ RegisterHTMLHandler(adaptor)
 
 const svg = new SVG({fontCache: "local"})
 
-const options = {
-  display: true,
-  em: 16,
-  ex: 8,
-  containerWidth: 80*16,
+type ConvertOptions = {
```

Comment:
This type (as well as `TexMacros`) should not be necessary to be duplicated here, as long as `mathjax.d.ts` is in scope. You just may need to use a namespace prefix to access those types. Also, is this TeX specific? If so, the name of this type should reflect this.